src/test/java/core/basesyntax/strategy/StrategyTest.java

```diff
@@ -0,0 +1,71 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.TransactionDto;
+import java.util.HashMap;
+import java.util.Map;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class StrategyTest {
+    private static Strategy strategy;
+
+    @BeforeClass
+    public static void beforeClass() throws Exception {
+        Map<String, OperationHandler> mapForStrategy = new HashMap<>();
+        mapForStrategy.put("b", new AddOperationHandler());
+        mapForStrategy.put("r", new AddOperationHandler());
+        mapForStrategy.put("s", new AddOperationHandler());
+        mapForStrategy.put("p", new SubtractOperationHandler());
+        strategy = new Strategy(mapForStrategy);
+    }
+
+    @Before
+    public void setUp() {
+        Storage.storage.clear();
+    }
+
+    @Test
+    public void balanceOperationTest_Ok() {
+        TransactionDto transaction
+                = new TransactionDto("b", new Fruit("banana"), 15);
+        strategy.getHandler(transaction).apply(transaction);
```

Comment:
you should test only strategy here - and each handler should be tested separately in its own class